catalogue/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -171,78 +381,23 @@ export const ConceptPage: NextPage<Props> = ({
         <ConceptImages as="section">
           <div className="container">
             <h2 className="h2 sectionTitle">Images</h2>
-
             {hasImagesTabs && (
               <TabNav
                 id="images"
                 selectedTab={selectedImagesTab}
                 variant="white"
-                items={[
-                  {
-                    id: 'images-about',
```

Comment:
Previously, we had three flavours of word separation in use.  

- hyphens: for the id used in tabNav
- camelCase: for the id used in the tab panel (also erroneously in aria-labelledby pointing to the tabNav)
- underscores: in the link source

I've reduced this to just the two.